src/components/Home-Languages.js

```diff
@@ -62,4 +62,16 @@ class HomeLanguages extends Component {
   }
 }
 
+// Java String hashcode implementation
```

Comment:
This needs to be inside the class and follow es6 format. e.g. `simpleHash = s => {...}`